### PR TITLE
Reduce jitter in name display

### PR DESCRIPTION
**Pull request by awushensky**

### CODE REVIEW

File: cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/result/OcrResultAggregator.kt [hunk 1]
```diff
@@ -83,7 +85,7 @@ class OcrResultAggregator(
             nameResults.countResult(name)
         } else 0
 
-        if (!isNameFound && nameCount >= 2) {
+        if (!isNameFound && requiredAgreementCount != null && nameCount >= requiredAgreementCount) {
```
Comment (xsl) [inline cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/result/OcrResultAggregator.kt:88]: should we have a separate `requiredAgreementCount` for PAN vs name?

File: cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/CardScanActivity.kt [hunk 2]
```diff
@@ -566,8 +566,8 @@ class CardScanActivity :
             fadeIn(cardPanTextView, 1.seconds)
         }
 
-        if (displayCardholderName && result.analyzerResult.name != null) {
-            cardNameTextView.text = result.analyzerResult.name
+        if (displayCardholderName && result.mostLikelyName != null) {
```
Comment (xsl) [inline cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/CardScanActivity.kt:569]: my original thinking was to show all of the intermediate names when it's debug mode, can we still preserve that?

Comment (awushensky) [reply cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/CardScanActivity.kt:569]: gotcha. we should tie that to debug then. I'll make that change.

File: cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/result/OcrResultAggregator.kt (hunk 1 above)
Comment (awushensky) [reply cardscan-ui/src/main/java/com/getbouncer/cardscan/ui/result/OcrResultAggregator.kt:88]: we certainly can. I'll separate.

**Pull request merged**